koku/masu/external/kafka_msg_handler.py

```diff
@@ -237,6 +237,19 @@ def construct_parquet_reports(request_id, context, report_meta, payload_destinat
     return daily_parquet_files
 
 
+def check_invalid_csv(dir_path, context):
```

Comment:
What is the purpose of this function? From what I can tell, all it does is log when we have an invalid report type. Does it need to remove the invalid files?

Reply:
I forgot to renamed the function. It's just going to log a warning message if the csv file is not a valid OCP report.